connectors/google/google-sheets/element-templates/google-sheets-outbound-connector.json

```diff
@@ -443,6 +443,12 @@
     "label" : "Row index",
     "description" : "Enter row index. Details in the <a href=\"https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/google-sheets/#what-is-a-row-index\" target=\"_blank\">documentation</a>",
     "optional" : true,
+    "constraints" : {
+      "notEmpty" : false,
```

Comment:
is this added by the template generator?

Reply:
yep